packages/ckeditor5-code-block/src/codeblockui.ts

```diff
@@ -64,7 +65,7 @@ export default class CodeBlockUI extends Plugin {
 
 			dropdownView.on( 'execute', evt => {
 				editor.execute( 'codeBlock', {
-					language: evt.source._codeBlockLanguage,
+					language: ( evt.source as { _codeBlockLanguage: string } )._codeBlockLanguage,
```

Comment:
Maybe we should consider adding typings for this `EventInfo`?

Reply:
We decided with @arkflpc to change it to `any`